scripts/test-upload

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# exit on error
+set -e
+# show commands as they are executed
+set -v
+
+[ $# -gt 0 ] && PIP_OPTS="$@"
+pip install --upgrade $PIP_OPTS setuptools wheel twine
```

Comment:
In that case, the makefile is also not portable, since it uses `python` instead of `python3`. Agreed that this should be changed, but it's not a release blocker. This script especially is only meant to run on the build server, since it needs my PyPI credentials.